app/helpers/contact_record_helper.rb

```diff
@@ -16,6 +16,8 @@ def message_heading(contact_record)
       "#{I18n.t("hub.messages.from")} #{contact_record.from}"
     when :outgoing_email
       "#{I18n.t("hub.messages.to")} #{contact_record.to}"
+    when :incoming_portal_message
```

Comment:
is it obvious that this is from the client if it just says from the client portal? Maybe redundant but would also mimic the structure of the other messages if included their name

Reply:
It will include their name! See screenshot. This is just what gets appended after the name.